package.json

```
@@ -12,11 +14,8 @@
   ],
   "scripts": {
     "bench": "node --no-warnings ./test/run-benchs.mjs",
-    "biome:format:fix": "biome format --fix ./",
-    "biome:format": "biome format ./",
-    "biome:lint:fix": "biome lint --fix ./",
-    "biome:lint": "biome lint ./",
-    "biome:ci": "biome ci ./",
+    "lint": "oxlint --config ./.oxlintrc.json5",
+    "lint:fix": "oxlint --config ./.oxlintrc.json5 --fix",
     "pjson:check": "node --no-warnings --test ./test/pjson.test.mjs",
     "pre-commit": "node --run biome:lint:fix; node --run biome:format:fix",
```

```suggestion
    "pre-commit": "node --run biome:lint:fix; node --run biome:format:fix",
```

remove or link as `lint:fix`